commit 062257c5d929e3c9a83a26624a09e57936ac6b5e
Author: Joseph Myers <josmyers@redhat.com>
Date:   Thu Dec 5 21:40:57 2024 +0000

    Fix typo in elf/Makefile:postclean-generated
    
    The postclean-generated setting in elf/Makefile lists
    $(objpfx)/dso-sort-tests-2.generated-makefile twice and
    $(objpfx)/dso-sort-tests-1.generated-makefile not at all, which looks
    like a typo; fix it to list each once.
    
    Tested for x86_64.

diff --git a/elf/Makefile b/elf/Makefile
index 96d5c290c9447c74..dfdcfc8e3d1e1770 100644
--- a/elf/Makefile
+++ b/elf/Makefile
@@ -1140,7 +1140,7 @@ $(objpfx)$(1).generated-makefile: $(1)
 endef
 endif
 
-postclean-generated += $(objpfx)/dso-sort-tests-2.generated-makefile \
+postclean-generated += $(objpfx)/dso-sort-tests-1.generated-makefile \
 		       $(objpfx)/dso-sort-tests-2.generated-makefile
 
 # Generate from each testcase description file
